src/file_worker.py

```diff
@@ -753,18 +736,21 @@ def get_dataset_file_infos(self, dataset_uuid, bearer_token=None):
             organ_dict['uuid'] = organ_uuid
 
             organ_info = self._get_entity(entity_id=organ_uuid, bearer_token=bearer_token, entity_type_check=['Sample'])
-            if not organ_info['sample_category'] or organ_info['sample_category'] != 'organ':
+            if not organ_info['sample_category'] or organ_info['sample_category'] != 'Organ':
```

Comment:
Not sure how SenNet stores the `sample_category` values in Neo4j. HuBMAP stores them as all lowercased values. We should convert the stored value to either all lowercase or all uppercase for comparison in this line otherwise it won't satisfy this condition.

Reply:
We base it on whatever UBKG provides us. So for sample category we base it off of this endpoint https://ontology.api.hubmapconsortium.org/valueset?parent_sab=SENNET&parent_code=C020076&child_sabs=SENNET

Reply:
This `organ_info` value is returned from entity-api, so the `organ_info['sample_category']` value will be the one from Neo4j and I can't see how the UBKG result would matter here.

Reply:
I misunderstood what you meant, added a fix https://github.com/hubmapconsortium/files-api/pull/37/commits/9ad3010f055c3a55bc69c78efcba5b1028303a52